www-apps/redmine/redmine-5.0.2.ebuild

```diff
@@ -0,0 +1,257 @@
+
+	if ! use imagemagick ; then
+		sed -i -e "/group :minimagick do/,/end$/d" Gemfile || die
+	fi
+	if ! use ldap ; then
+		# remove ldap stuff module if disabled to avoid #413779
+		use ldap || rm app/models/auth_source_ldap.rb || die
+		sed -i -e "/group :ldap do/,/end$/d" Gemfile || die
+	fi
+	if ! use markdown ; then
+		sed -i -e "/group :markdown do/,/end$/d" Gemfile || die
+	fi
+	# Additional dependency for Gemfile (#657156)
+	if use fastcgi; then
+		echo "gem 'fcgi'" > Gemfile.local
+	fi
+}
+
+all_ruby_install() {
+	dodoc doc/* README.rdoc
+	rm -r doc test appveyor.yml CONTRIBUTING.md README.rdoc || die
+
+	keepdir /var/log/${PN}
+
+	insinto "${REDMINE_DIR}"
+	doins -r .
+	insinto "${REDMINE_DIR}/config"
+	doins "${FILESDIR}/additional_environment.rb"
+	keepdir "${REDMINE_DIR}"/{app/views/previews,files,public/plugin_assets,vendor}
+	keepdir "${REDMINE_DIR}"/tmp/{cache,imports,sessions,sockets}
+
+	fowners -R redmine:redmine \
+		"${REDMINE_DIR}/config.ru" \
+		"${REDMINE_DIR}/config" \
+		"${REDMINE_DIR}/files" \
+		"${REDMINE_DIR}/public/plugin_assets" \
+		"${REDMINE_DIR}/tmp" \
+		/var/log/${PN}
+
+	fowners redmine:redmine "${REDMINE_DIR}"
+
+	# protect sensitive data, see bug #406605
+	fperms -R go-rwx \
+		"${REDMINE_DIR}/config" \
+		"${REDMINE_DIR}/files" \
+		"${REDMINE_DIR}/tmp" \
+		/var/log/${PN}
+
+	if use passenger; then
+		has_apache
+		insinto "${APACHE_VHOSTS_CONFDIR}"
+		doins "${FILESDIR}/10_redmine_vhost.conf"
+	fi
+
+	# logrotate
+	insinto /etc/logrotate.d
+	newins "${FILESDIR}"/redmine.logrotate redmine
+
+	newconfd "${FILESDIR}/${PN}.confd" ${PN}
+	newinitd "${FILESDIR}/${PN}-3.initd" ${PN}
+	doenvd "${T}/50${PN}"
+}
+
+pkg_postinst() {
+	if [[ -e "${EROOT}/${REDMINE_DIR}/config/initializers/session_store.rb" \
+	|| -e "${EROOT}/${REDMINE_DIR}/config/initializers/secret_token.rb" ]]
+	then
+		elog "Execute the following command to upgrade environment:"
+		elog
+		elog "# emerge --config \"=${CATEGORY}/${PF}\""
+		elog
+		elog "For upgrade instructions take a look at:"
+		elog "http://www.redmine.org/wiki/redmine/RedmineUpgrade"
+	else
+		elog "Execute the following command to initialize environment:"
+		elog
+		elog "# cd ${EROOT}/${REDMINE_DIR}"
+		elog "# cp config/database.yml.example config/database.yml"
+		elog "# \${EDITOR} config/database.yml"
+		elog "# chown redmine:redmine config/database.yml"
+		elog "# emerge --config \"=${CATEGORY}/${PF}\""
+		elog
+		elog "Installation notes are at official site"
+		elog "http://www.redmine.org/wiki/redmine/RedmineInstall"
+	fi
+
+	elog
+	elog "OpenID support was removed in Redmine 5.0. If you are using OpenID"
+	elog "authentication, you should switch back to Redmine 4.2 or install and"
+	elog "enable an OpenID plugin. See https://redmine.org/issues/35755."
+	elog
+}
+
+pkg_config() {
+	# Remove old lock file
+	rm -f "${EROOT}/${REDMINE_DIR}/Gemfile.lock"
+
+	if [[ ! -e "${EROOT}/${REDMINE_DIR}/config/database.yml" ]]; then
```

Comment:
Since your `${REDMINE_DIR}` variable contains a full path with `/...` you have 2 slashes here, and basically everywhere in this ebuild that calls `${EROOT}/${REDMINE_DIR}`. It should still work but you may want to fix it? :)

Rest looks good.

Reply:
Fixed.
Additionally add missed dependency dev-ruby/sanitize